src/dashboard/ChartDashboardObject.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace luya\admin\dashboard;
+
+use Yii;
+use yii\helpers\Html;
+use yii\helpers\ArrayHelper;
+
+/**
+ * Fast generated Dashboard Objects.
+ *
+ * The default object is the default class for all {{luya\admin\base\Module::$dashboardObjects}} items without a class defintion.
+ *
+ * @author Oom <baqianxin@163.com>
+ * @since 1.0.2
+ * Example usage:
```

Comment:
PHPDoc should be always at the end. Please, again, take a look at this example, i am sure you will see the difference => https://github.com/luyadev/luya/blob/master/core/Hook.php